.gitignore

```diff
@@ -141,4 +141,9 @@ dmypy.json
 # vscode folder
 .vscode/
 
+# avoid automatic file add into git commit
+configs/
```

Comment:
why is `configs/` in here?

Reply:
added exceptions for the example files:

# Except the following files
!configs/example-config-extpipesv2.yml
!configs/example-config-extpipesv2to3.yml
!configs/example-config-extpipesv3.yml
!configs/example-config-extpipesv3-minimum.yml

Reply:
updated to 

!configs/example-config-extpipes*